apps/ledger-live-desktop/src/renderer/modals/Send/fields/AmountField.tsx

```diff
@@ -14,16 +12,16 @@ import Switch from "~/renderer/components/Switch";
 import Text from "~/renderer/components/Text";
 
 type Props = {
-  parentAccount: ?Account,
-  account: AccountLike,
-  transaction: Transaction,
-  onChangeTransaction: (*) => void,
-  status: TransactionStatus,
-  bridgePending: boolean,
-  t: TFunction,
-  initValue?: BigNumber,
-  walletConnectProxy?: boolean,
-  resetInitValue?: () => void,
+  parentAccount?: Account;
+  account: AccountLike;
+  transaction: Transaction;
+  onChangeTransaction: (_: any) => void;
```

Comment:
Avoid `any` as well here please 🙏

Reply:
I don't like using any but everywhere else this function is called the parameters are of type `any`

Reply:
No it's not, actually you can see here the real type of `onChangeTransaction`: https://github.com/LedgerHQ/ledger-live/blob/7806de7ef3f402e1b87df8ac635c49d00c832fff/apps/ledger-live-desktop/src/renderer/modals/Send/types.js#L27